basic-api.yml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.0
+info:
+  description: This is an open HTTP API specification for issuing Verifiable Credentials based on the W3C Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/). Implementations of this API may differ in terms of which representation formats or proof formats they do or do not support. Implementations MAY choose to not support certain parts of this API (e.g. optional parts include changing and retrieving the status of a credential, or refreshing a credential).
+  version: "0.0.2"
+  title: Verifiable Credential Issuer HTTP API
+  contact:
+    email: msporny@digitalbazaar.com
+  license:
+    name: All documents in this Repository are licensed by contributors under the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
+paths:
+  /credentials:
+    post:
+      summary: Issues a verifiable credential and returns it in the response body.
+      operationId: issueCredential
+      description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueRequest'
+        description: Parameters for issuing the credential.
+      responses:
+        '201':
+          description: Credential was successfully issued.
+          content:
+            application/ld+json:
```

Comment:
The API response should not be restricted to JSON-LD.

Reply:
> The API response should not be restricted to JSON-LD.

I agree, but when I typed out the application/json response, it looked exactly the same. Do you want me to just copy-paste the application/json response?